src/walletHandlers/verifyHandler.js

```
@@ -27,72 +30,73 @@ module.exports = async (from_address, data) => {
-            const signedData = JSON.parse(signed_message);
-
-            const { address, provider, ...data } = signedData;
+            const signedData = JSON.parse(signed_message.trim());
+            const { message, ...data } = signedData;
+            let attestationWalletAddress = data.address;
 
-            if (walletAddress || !address) {
-                if (walletAddress === address) {
-                    const order = await DbService.getAttestationOrders({ serviceProvider: provider, data, address });
+            // get and validate address
+            if (message && message.includes('I own the address:')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)